lib/super_digit.rb

```diff
@@ -1,15 +1,33 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n log n) where n is the value of the passed in integer (and thus determines if the helper will be called... uh idk why I put "n" as the coefficient I just just know it is a variable that will determine if the helper method is called, potentially multiple times...)
+# Space Complexity - O(n log n)
 def super_digit(n)
```

Comment:
The time complexity is O(n^2) where n is the number of digits, or O( (log n)^2) where n is the given parameter.